README.md

```diff
@@ -646,6 +646,7 @@ Table of Contents
 
   * [algolia.com](https://www.algolia.com/) — Hosted search-as-you-type (instant). Free hacker plan up to 10,000 documents and 100,000 operations. Bigger free plans available for community/Open Source projects
   * [bonsai.io](https://bonsai.io/) — Free 1 GB memory and 1 GB storage
+  * [MongoDB Atlas Search](https://www.mongodb.com/atlas/search) — Free 512 GB of storage, 3 search indices, search-as-you-type (optional), unlimited operations.
```

Comment:
We already have an atlas entry here, perhaps we can expand that one instead of adding a 2nd entry